opennms-base-assembly/src/main/filtered/etc/opennms.properties

```diff
@@ -710,6 +711,20 @@ excludeServiceMonitorsFromRemotePoller=DHCP,NSClient,RadiusAuth,XMP
 #Disables the processing of counter wraps, replacing these with NaNs instead.
 org.opennms.newts.nan_on_counter_wrap=true
 
+###### Timeseries Integration Layer #####
+# Use these properties to configure persistence using the new timeseries abstraction layer
+# Note that timeseries must be enabled using the 'org.opennms.timeseries.strategy' property
+# for these to take effect. org.opennms.timeseries.strategy=integration
+
+# How big should a batch be that is passed to the timeseries implementation?
```

Comment:
Let's remove these and just reference the documentation for now. (Trying to avoid users confusing these properties with those provided by Newts.)

Reply:
ok, done.